stdlib/join/join_fn.go

```diff
@@ -211,6 +211,22 @@ func (f *JoinFn) createSchema(record values.Object) ([]flux.ColMeta, error) {
 			Type:  ty,
 		})
 	}
+	for _, gcol := range groupkey.Cols() {
+		found := false
+		for _, col := range cols {
+			if gcol == col {
+				found = true
```

Comment:
We could `break` after here. Although, if cols is sorted would a quick binary search be worthwhile?